.tekton/konflux-e2e-tests-pull-request.yaml

```diff
@@ -7,8 +7,10 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "main"
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request"
+      && target_branch == "main"
+      && files.all.exists(x, !x.matches('^(?:docs|examples|enhancements|contrib)/|\\.md$|^(?:.*/)?(?:\\.gitignore|OWNERS|PROJECT|LICENSE|.golangci.yml|.sonarcloud.properties)$'))
```

Comment:
can we add directories
* `.github`
*  `.vscode`
* `pipelines`
* `templates` 
 
and files 
* `OWNERS_ALIASES`
* `.dockerignore`

(the same for the rule in `.tekton/konflux-e2e-tests-push.yaml`)

Reply:
done.